ununifi/chain.json

```diff
@@ -0,0 +1,174 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "ununifi",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://ununifi.io/",
+  "pretty_name": "UnUniFi",
+  "chain_id": "ununifi-beta-v1",
+  "bech32_prefix": "ununifi",
+  "daemon_name": "guu",
+  "node_home": "$HOME/.ununifid",
+  "key_algos": ["secp256k1"],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "uguu",
+        "fixed_min_gas_price": 0.0025,
+        "low_gas_price": 0.0025,
+        "average_gas_price": 0.025,
+        "high_gas_price": 0.04
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "uguu"
+      }
+    ],
+    "lock_duration": {
+      "time": "1209600s"
+    }
+  },
+  "codebase": {
+    "git_repo": "https://github.com/UnUniFi/chain",
+    "recommended_version": "v2.0.0",
+    "compatible_versions": ["v2.0.0"],
+    "binaries": {
+      "linux/amd64": "https://github.com/UnUniFi/chain/releases/download/v2.0.0/ununifid"
+    },
+    "cosmos_sdk_version": "0.47",
+    "consensus": {
+      "type": "cometbft",
+      "version": "0.34"
+    },
+    "cosmwasm_version": "0.40",
+    "cosmwasm_enabled": true,
+    "ibc_go_version": "7.0.0",
+    "ics_enabled": ["ics20-1"],
```

Comment:
can we please copy of this codebase data into the v2.0.0 version object so that upcoming automation doesn't overwrite it?

Reply:
@JeremyParish69 Copied and added. Updated the versions as the latest version is now v2.1.0.

Reply:
Thanks for this! Although the latest version must go last in the array. Can you reverse the order of those version in the array please?

Reply:
@JeremyParish69 Changed. I hope this is all right.